### ISSUE TITLE
[Issue]: Not found Jellyfin-web

### ISSUE THREAD

**Issue by AngelBrielez** [jellyfin/jellyfin#8383]
### Please describe your bug

The client side suddenly stopped working. I think it may be an update that broke it and apparently it can't find the Jellyfin-web.

### Jellyfin Version

Other

### if other:

10.8.4-1

### Environment

```markdown
- OS: Linux
- Clients: Firetv, Android, Ubuntu
- Browser: Firefox
- 64-Bit Process: True
- User Interactive: True
- Processor count: 12
- Program data path: /home/angelo/.local/share/jellyfin
- Web resources path: /usr/lib/jellyfin/bin/jellyfin-web
```

### Jellyfin logs

```shell
[09:34:46] [INF] [1] Main: Application directory: /usr/lib/jellyfin/bin/
[09:34:46] [ERR] [1] Main: The server is expected to host the web client, but the provided content directory is either invalid or empty: /usr/lib/jellyfin/bin/jellyfin-web. If you do not want to host the web client with the server, you may set the '--nowebclient' command line flag, or set'hostwebclient=false' in your config settings.
[09:34:46] [INF] [2] Main: Received a SIGTERM signal, shutting down
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Bond-009** [comment]
Can you give some more info, installation method, how old your install is (first installed version), etc

**Comment by AngelBrielez** [comment]
> Can you give some more info, installation method, how old your install is (first installed version), etc

¡Yes! the installation method is: Ubuntu, Stable.
![imagen](https://user-images.githubusercontent.com/39037416/190746902-c4b99c4d-5fd9-4ceb-a844-f2680a4da586.png)

The version of Jellyfin is: Jellyfin.Server 10.8.4.0, I have no idea about the previous version, the server broke maybe in an update.

It appears that the jellyfin web has been deleted as indicated in the log.

[09:34:46] [INF] [1] Main: Application directory: /usr/lib/jellyfin/bin/
[09:34:46] [ERR] [1] Main: The server is expected to host the web client, but the provided content directory is either invalid or empty: /usr/lib/jellyfin/bin/jellyfin-web. If you do not want to host the web client with the server, you may set the '--nowebclient' command line flag, or set'hostwebclient=false' in your config settings.
[09:34:46] [INF] [2] Main: Received a SIGTERM signal, shutting down.

I will try, installing again.

**Comment by AngelBrielez** [comment]
Not working yet u.u

**Comment by crobibero** [comment]
Try running `apt install jellyfin-web`

**Comment by beren12** [comment]
for some reason the web installs jellyfin-web to /usr/share/jellyfin/web so a symlink needs to be made, or the deb install location updated to /usr/lib/jellyfin/bin/jellyfin-web. BTW auto-closing bugs without fixing them is wishful thinking, at best.

**Comment by beren12** [comment]
This is still a bug 14 months later.

**Comment by Reezlaw** [comment]
This just happened to me as well after an update on Manjaro from AUR. Reinstalling both from AUR and from the official repo (extra) did not fix it.

**Comment by mhalano** [comment]
Yeah. I just upgraded to 10.9 and got this problem. I can see the correct path, `/usr/share/jellyfin/web`, is set as an argument for the service in systemd, but for some reason it looks for this old path that seems cached in some place, even I remove the server with all config files and install again.

**Comment by Helium-7** [comment]
yeah so i just had this too, been unsuccessfully trying to fix it for days now
got it running on a barebones arch install ~~because i like to torture myself~~ and been using it mostly without problems for about a year now, just updated the system, because at first, only the website wouldnt load, but finamp (music client for android) still worked fine, after the update, it didnt work anymore either and the systemd process kept failing to start, if i tried to manually start it, i'd get the same error about missing `jellyfin-web` as above, and the logs showed the same..
I went to look for the folder ofc, or where it's supposed to be, and it was just gone or maybe never there in the first place? 
If just the install location changed with an update, i cant find the old location anywhere..
`[23:09:41] [INF] [1] Main: Jellyfin version: 10.9.3
[23:09:41] [INF] [1] Main: Environment Variables: ["[JELLYFIN_LOG_DIR, /home/ruby/.local/share/jellyfin/log]"]
[23:09:41] [INF] [1] Main: Arguments: ["/usr/lib/jellyfin/jellyfin.dll"]
[23:09:41] [INF] [1] Main: Operating system: Arch Linux
[23:09:41] [INF] [1] Main: Architecture: X64
[23:09:41] [INF] [1] Main: 64-Bit Process: True
[23:09:41] [INF] [1] Main: User Interactive: True
[23:09:41] [INF] [1] Main: Processor count: 4
[23:09:41] [INF] [1] Main: Program data path: /home/ruby/.local/share/jellyfin
[23:09:41] [INF] [1] Main: Log directory path: /home/ruby/.local/share/jellyfin/log
[23:09:41] [INF] [1] Main: Config directory path: /home/ruby/.config/jellyfin
[23:09:41] [INF] [1] Main: Cache path: /home/ruby/.cache/jellyfin
[23:09:41] [INF] [1] Main: Web resources path: /usr/lib/jellyfin/jellyfin-web
[23:09:41] [INF] [1] Main: Application directory: /usr/lib/jellyfin/
[23:09:41] [ERR] [1] Main: The server is expected to host the web client, but the provided content directory is either invalid or empty: /usr/lib/jellyfin/jellyfin-web. If you do not want to host the web client with the server, you may set the '--nowebclient' command line flag, or set'hostwebclient=false' in your config settings
`

**Comment by silajim** [comment]
How is this thing not fixed still? It just happened to me on Ubuntu

**Comment by Helium-7** [comment]
> How is this thing not fixed still? It just happened to me on Ubuntu

I eventually fixed it, see [my thread on the jellyfin forum](https://forum.jellyfin.org/t-solved-updating-jellyfin-on-arch-not-working)
It required uninstalling and installing jellyfin anew, users don't survive it, plugins dont seem to either, nor the metadata, but everthing else seems to.

**Comment by Helium-7** [comment]
This [second issue](https://forum.jellyfin.org/t-solved-jellyfin-web-folder-missing), which is actually why i found this github thread, then occurred later on. I believe part of the reason for the issue was that jellyfin moved repository, it is no longer on the AUR, but on the official arch extras repo, however, that's only relevant for the arch version of it ofc.

**Comment by silajim** [comment]
I did remove it and uninstall it, but it happened again.

**Comment by ACR-Jeff** [comment]
> for some reason the web installs jellyfin-web to /usr/share/jellyfin/web so a symlink needs to be made, or the deb install location updated to /usr/lib/jellyfin/bin/jellyfin-web. BTW auto-closing bugs without fixing them is wishful thinking, at best.

I am not understanding this. Do I create a symlink to /usr/share/jellyfin/web directory or the contents within to /usr/lib/jellyfin/bin/jellyfin-web

If someone could help would be greatly appreciated, Thanks

**Comment by ACR-Jeff** [comment]
Ok I figured the symlink out `sudo ln -s /usr/share/jellyfin/web /usr/lib/jellyfin/bin/jellyfin-web`
Yet still facing the exact same issue

**Comment by ACR-Jeff** [comment]
Ok so I was finely able to login after the symlink with my admin account, The issue seems to now be the fact it doesn't except public ip's, I was only able to login from the server itself via jellyfin-web 127.0.0.1:8096

**Comment by ACR-Jeff** [comment]
For the time being I solved my issue, After logging in I checked out the logs, Any LAN Network IP I have added in LAN Networks from previous versions if it didn't have a /24 subnet behind the LAN ip addresses it was automatically set to /32 subnet which prevents connection to any hosts and changes it's subnet to an incorrect subnet. Adding /24 behind those addresses connected instantly. 
In Dashboard > Networking > LAN Network
Add your LAN Network IP's example: 192.168.1.155/24, 192.168.1.266/24

Jellyfin Logs for anyone interested:
[2024-10-07 15:19:32.994 -07:00] [INF] Defined LAN subnets: ["127.0.0.1/24", "192.168.1.0/24", "192.168.1.1/24", "192.168.1.177/32", "192.168.1.2/32", "192.168.1.22/32", "192.168.1.169/24", "192.168.1.12/32", "192.168.1.143/32", "192.168.1.225/32", "192.168.1.144/24"]
[2024-10-07 15:19:32.995 -07:00] [INF] Defined LAN exclusions: ["192.168.1.177/32", "192.168.1.2/32", "192.168.1.22/32", "192.168.1.12/32", "192.168.1.143/32", "192.168.1.225/32"]
[2024-10-07 15:19:32.995 -07:00] [INF] Used LAN subnets: ["127.0.0.1/24", "192.168.1.0/24", "192.168.1.1/24", "192.168.1.177/32", "192.168.1.2/32", "192.168.1.22/32", "192.168.1.169/24", "192.168.1.12/32", "192.168.1.143/32", "192.168.1.225/32", "192.168.1.144/24"]
[2024-10-07 15:19:32.995 -07:00] [INF] Filtered interface addresses: ["127.0.0.1", "192.168.1.177"]
[2024-10-07 15:19:32.995 -07:00] [INF] Bind Addresses ["0.0.0.0"]
[2024-10-07 15:19:32.995 -07:00] [INF] Remote IP filter is "Allowlist"
[2024-10-07 15:19:32.995 -07:00] [INF] Filtered subnets: []
[2024-10-07 15:20:35.040 -07:00] [INF] Authentication request for "Media-Server" has succeeded.
[2024-10-07 15:20:35.040 -07:00] [INF] Current/Max sessions for user "Media-Server": 0/0
[2024-10-07 15:20:35.040 -07:00] [INF] Creating new access token for user 66d78bab-bf38-41c3-b5e4-264044c5508a
[2024-10-07 15:20:35.103 -07:00] [INF] WS "192.168.1.144" request
[2024-10-07 15:21:45.201 -07:00] [INF] WS "192.168.1.169" request
[2024-10-07 15:22:11.291 -07:00] [INF] Defined LAN subnets: ["127.0.0.1/24", "192.168.1.0/24", "192.168.1.1/24", "192.168.1.177/24", "192.168.1.2/32", "192.168.1.22/32", "192.168.1.169/24", "192.168.1.12/32", "192.168.1.143/32", "192.168.1.225/32", "192.168.1.144/32"]
[2024-10-07 15:22:11.291 -07:00] [INF] Defined LAN exclusions: ["192.168.1.2/32", "192.168.1.22/32", "192.168.1.12/32", "192.168.1.143/32", "192.168.1.225/32", "192.168.1.144/32"]
[2024-10-07 15:22:11.291 -07:00] [INF] Used LAN subnets: ["127.0.0.1/24", "192.168.1.0/24", "192.168.1.1/24", "192.168.1.177/24", "192.168.1.2/32", "192.168.1.22/32", "192.168.1.169/24", "192.168.1.12/32", "192.168.1.143/32", "192.168.1.225/32", "192.168.1.144/32"]
[2024-10-07 15:22:11.291 -07:00] [INF] Filtered interface addresses: ["127.0.0.1", "192.168.1.177"]
[2024-10-07 15:22:11.291 -07:00] [INF] Bind Addresses ["0.0.0.0"]
[2024-10-07 15:22:11.291 -07:00] [INF] Remote IP filter is "Allowlist"
[2024-10-07 15:22:11.291 -07:00] [INF] Filtered subnets: []
[2024-10-07 15:22:19.634 -07:00] [WRN] WS "192.168.1.144" error receiving data: "The remote party closed the WebSocket connection without completing the close handshake."
[2024-10-07 15:22:19.639 -07:00] [INF] WS "192.168.1.144" closed
[2024-10-07 15:23:04.198 -07:00] [INF] WS "192.168.1.169" closed

**Comment by Helium-7** [comment]
> For the time being I solved my issue, After logging in I checked out the logs, Any LAN Network IP I have added in LAN Networks from previous versions if it didn't have a /24 subnet behind the LAN ip addresses it was automatically set to /32 subnet which prevents connection to any hosts and changes it's subnet to an incorrect subnet. Adding /24 behind those addresses connected instantly, This wasn't the case in previous versions of Jellyfin. In Dashboard > Networking > LAN Network Add your LAN Network IP's example: 192.168.1.155/24, 192.168.1.266/24
> 
> Jellyfin Logs for anyone interested: [2024-10-07 15:19:32.994 -07:00] [INF] Defined LAN subnets: ["127.0.0.1/24", "192.168.1.0/24", "192.168.1.1/24", "192.168.1.177/32", "192.168.1.2/32", "192.168.1.22/32", "192.168.1.169/24", "192.168.1.12/32", "192.168.1.143/32", "192.168.1.225/32", "192.168.1.144/24"] [2024-10-07 15:19:32.995 -07:00] [INF] Defined LAN exclusions: ["192.168.1.177/32", "192.168.1.2/32", "192.168.1.22/32", "192.168.1.12/32", "192.168.1.143/32", "192.168.1.225/32"] [2024-10-07 15:19:32.995 -07:00] [INF] Used LAN subnets: ["127.0.0.1/24", "192.168.1.0/24", "192.168.1.1/24", "192.168.1.177/32", "192.168.1.2/32", "192.168.1.22/32", "192.168.1.169/24", "192.168.1.12/32", "192.168.1.143/32", "192.168.1.225/32", "192.168.1.144/24"] [2024-10-07 15:19:32.995 -07:00] [INF] Filtered interface addresses: ["127.0.0.1", "192.168.1.177"] [2024-10-07 15:19:32.995 -07:00] [INF] Bind Addresses ["0.0.0.0"] [2024-10-07 15:19:32.995 -07:00] [INF] Remote IP filter is "Allowlist" [2024-10-07 15:19:32.995 -07:00] [INF] Filtered subnets: [] [2024-10-07 15:20:35.040 -07:00] [INF] Authentication request for "Media-Server" has succeeded. [2024-10-07 15:20:35.040 -07:00] [INF] Current/Max sessions for user "Media-Server": 0/0 [2024-10-07 15:20:35.040 -07:00] [INF] Creating new access token for user 66d78bab-bf38-41c3-b5e4-264044c5508a [2024-10-07 15:20:35.103 -07:00] [INF] WS "192.168.1.144" request [2024-10-07 15:21:45.201 -07:00] [INF] WS "192.168.1.169" request [2024-10-07 15:22:11.291 -07:00] [INF] Defined LAN subnets: ["127.0.0.1/24", "192.168.1.0/24", "192.168.1.1/24", "192.168.1.177/24", "192.168.1.2/32", "192.168.1.22/32", "192.168.1.169/24", "192.168.1.12/32", "192.168.1.143/32", "192.168.1.225/32", "192.168.1.144/32"] [2024-10-07 15:22:11.291 -07:00] [INF] Defined LAN exclusions: ["192.168.1.2/32", "192.168.1.22/32", "192.168.1.12/32", "192.168.1.143/32", "192.168.1.225/32", "192.168.1.144/32"] [2024-10-07 15:22:11.291 -07:00] [INF] Used LAN subnets: ["127.0.0.1/24", "192.168.1.0/24", "192.168.1.1/24", "192.168.1.177/24", "192.168.1.2/32", "192.168.1.22/32", "192.168.1.169/24", "192.168.1.12/32", "192.168.1.143/32", "192.168.1.225/32", "192.168.1.144/32"] [2024-10-07 15:22:11.291 -07:00] [INF] Filtered interface addresses: ["127.0.0.1", "192.168.1.177"] [2024-10-07 15:22:11.291 -07:00] [INF] Bind Addresses ["0.0.0.0"] [2024-10-07 15:22:11.291 -07:00] [INF] Remote IP filter is "Allowlist" [2024-10-07 15:22:11.291 -07:00] [INF] Filtered subnets: [] [2024-10-07 15:22:19.634 -07:00] [WRN] WS "192.168.1.144" error receiving data: "The remote party closed the WebSocket connection without completing the close handshake." [2024-10-07 15:22:19.639 -07:00] [INF] WS "192.168.1.144" closed [2024-10-07 15:23:04.198 -07:00] [INF] WS "192.168.1.169" closed

you should be able to leave that field empty, unless you wish to block IPs from your local network from access. 

> Comma separated list of IP addresses or IP/netmask entries for networks that will be considered on local network when enforcing bandwidth restrictions. If set, all other IP addresses will be considered to be on the external network and will be subject to the external bandwidth restrictions. If left blank, only the server's subnet is considered to be on the local network.

"the server's subnet" just needs to be set correctly, which is /24 for most private networks, just specifying one IP without a subnet will make it a network with just one IP, also known as /32. This should've been the case in previous versions too, as that is what the description of the setting says..
